content/tutorials/r_inboveg/index.Rmd

```diff
@@ -263,7 +276,7 @@ allecodes
 ## Qualifiers information
 
 This function `inboveg_qualifiers`queries the INBOVEG database for
-qualifier information on recordings for one or more surveys. These qualifiers 
+qualifier information (site and/or management) on recordings for one or more surveys. These qualifiers 
```

Comment:
Should also mention that "management qualifier" = MQ and the other is SQ.

Reply:
added this extra information